system-tests/projects/migration-e2e-no-plugins-support-file/cypress.json

```diff
@@ -1 +1,4 @@
-{}
```

Comment:
Did we need to update a test to verify this configuration was correct in the migration workflow?